### ISSUE TITLE
Implement more secure AppId

### ISSUE THREAD

**Issue by hthetiot** [TelegramPassport/passport-js#2]
Currently AppId generated on https://telepass.me are sequential (10003, 10004) and can allow attacker to simply write script to list all client Id.

Possible solution: 
- Generate UUID
- Apply XOR on AppId with a secret salt

**Comment by hthetiot** [comment]
Pfff even Oauth spec missed that:
- https://tools.ietf.org/html/rfc6749#section-2.2

**Comment by ingria** [comment]
Yeah, the initial idea was using UUIDs for both app_secret and app_id, but it's a double-edged sword:

Pros:
+ Higher entropy (meaning it's nearly impossible to bruteforce)
+ RFC [recommends](https://tools.ietf.org/html/rfc6749#page-71) it to be VSCHAR

Cons:
- Poor perfomance (non-sequential values cannot be indexed)
- It's generally pointless (because it is public and can be seen by anyone, so why bother?)

So, I chose better perfomance over questionable security.

By the way, Telegram uses plain INT for app_id too. Check it here: https://my.telegram.org/apps
